client-app/core/composables/useHotjar.ts

```diff
@@ -0,0 +1,47 @@
+import { canUseDOM } from "@apollo/client/utilities";
+import Hotjar from "@hotjar/browser";
+import { useThemeContext } from "@/core/composables/useThemeContext";
+import { IS_DEVELOPMENT } from "@/core/constants";
+import { Logger } from "@/core/utilities";
+
+const { modulesSettings } = useThemeContext();
+
+const DEBUG_PREFIX = "[Hotjar]";
+
+const MODULE_KEYS = {
+  ID: "VirtoCommerce.Hotjar",
+  ENABLE_STATE: "Hotjar.EnableTracking",
+  TRACK_ID: "Hotjar.SiteId",
+  SNIPPET_VERSION: "Hotjar.SnippetVersion",
+};
+
+function init() {
+  if (!canUseDOM) {
+    return;
+  }
+  const moduleSettings = modulesSettings?.value?.find((el) => el.moduleId === MODULE_KEYS.ID);
+
+  const [isEnabled, id, version] = [
+    !!getSettingsValue(moduleSettings?.settings, MODULE_KEYS.ENABLE_STATE),
+    Number(getSettingsValue(moduleSettings?.settings, MODULE_KEYS.TRACK_ID)),
+    Number(getSettingsValue(moduleSettings?.settings, MODULE_KEYS.SNIPPET_VERSION)),
```

Comment:
Number(undefined) or Number(null) generate falsy  value but correct type.
Need for check above
TRACK_ID is string, but need nuber

Reply:
So why have this implicit behavior instead of correct types as
```typescript
const idValue = getSettingsValue(moduleSettings?.settings, MODULE_KEYS.TRACK_ID)
const id = isDefined(idValue) ? Number(idValue) : null;
const version = getSettingsValue(moduleSettings?.settings, MODULE_KEYS.SNIPPET_VERSION) as Number | null | undefined;
```
or even
```typescript
const id = getSettingsValue(moduleSettings?.settings, MODULE_KEYS.TRACK_ID) as Number | null | undefined;
const version = getSettingsValue(moduleSettings?.settings, MODULE_KEYS.SNIPPET_VERSION) as Number | null | undefined;
```
if backend will be fixed?